src/olympia/scanners/admin.py

```diff
@@ -239,7 +254,8 @@ def formatted_matched_rules(self, obj):
     formatted_matched_rules.short_description = 'Matched rules'
 
     def handle_true_positive(self, request, pk, *args, **kwargs):
-        if request.method != "POST":
+        is_admin = acl.action_allowed(request, amo.permissions.ADMIN_ADVANCED)
+        if not is_admin or request.method != "POST":
```

Comment:
There was no check previously because only admins could get to this page (behind a VPN). Adding a new permission made me think that now would be a good time to actually secure those actions, too.